static/src/javascripts/projects/common/modules/experiments/tests/giraffe.js

```diff
@@ -0,0 +1,63 @@
+define([
+    'common/utils/$',
+    'common/utils/template',
+    'text!common/views/giraffe-message.html',
+    'common/views/svgs'
```

Comment:
You can import the inline SVG directly with `'inlineSvg!svgs/icon/arrowWhiteRight',` and import `common/views/svg` (no s) to enable you to add the class - this will avoid importing all the svgs